src/components/content/horizon.tsx

```diff
@@ -3,21 +3,22 @@ import styled from 'styled-components'
 
 import { makePadding } from '../../helper/css'
 
-interface HorizonEntry {
-  imageUrl: string
+export interface HorizonEntry {
   title: string
+  imageUrl: string
   text: string
   url: string
+  frequency: number
 }
 
-interface HorizonProps {
+export interface HorizonProps {
   entries: HorizonEntry[]
 }
 
 export function Horizon({ entries }: HorizonProps) {
   return (
     <Wrapper>
-      {entries.map((horizonEntry, index) => {
+      {entries.slice(0, 3).map((horizonEntry, index) => {
```

Comment:
out of curiosity: is there any case where entries are longer then 3?

Reply:
nope, leftover from when I tried to implement the logic in the component. 
removed now.